ksp_plugin_adapter/ksp_plugin_adapter.cs

```diff
@@ -223,29 +221,43 @@ private enum UnityLayers {
   private Dictionary<uint, QP> part_id_to_degrees_of_freedom_ =
       new Dictionary<uint, QP>();
 
-  // The first apocalyptic error message.
+  // UI for the apocalypse notification.
   [KSPField(isPersistant = true)]
   private String revelation_ = "";
-  // Whether we have encountered an apocalypse already.
   [KSPField(isPersistant = true)]
   private bool is_post_apocalyptic_ = false;
```

Comment:
Keep the comments that document the semantics of the string and bool.

Reply:
Done.